OneCommunityGlobal/HGNRest
# Huijie - fix ability edit start and end date

# Description
![task description](https://github.com/user-attachments/assets/ce527a00-1ac9-476b-b9fd-37576ebbba96)

## Related PRS (if any):
This backend PR is related to the #2655 frontend PR (https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2655).

## Main changes explained:
1. When the accounts start date or end date is being saved, assume that the entered date on userprofile page is in 'America/Los_Angeles' time zone. This will prevent some time zone errors.


## How to test:
1. check into current branch
2. do `npm install`, `npm run build` and `npm start` to run this PR locally
3. clear site data/cache
4. log as an owner or admin user
5. Please check #2655 frontend PR for more details. Thank you!
